cmd/csi-addons/main.go

```diff
@@ -56,6 +59,9 @@ func init() {
 	flag.StringVar(&cmd.operation, "operation", "", "csi-addons operation")
 	flag.StringVar(&cmd.persistentVolume, "persistentvolume", "", "name of the PersistentVolume")
 	flag.StringVar(&cmd.drivername, "drivername", "", "name of the CSI driver")
+	flag.StringVar(&cmd.secretName, "secretname", "", "name of the kubernetes secret")
```

Comment:
I would prefer to have a single option `--secret=namespace/secretname`, you can easily split the argument on the `/`, and have it a little more convenient for users of the tool. Potentially the namespace could be optional, and you can then assume the namespace is the one where the container is running (maybe you can get that from the environment?).

Reply:
Taking the secret details in a single option now as `namespace/name` and we also need to pass the clusterID as Parameter for (un)fenceing.